block/olya-iosched.c

```diff
@@ -94,6 +90,8 @@ static int noop_init_queue(struct request_queue *q, struct elevator_type *e)
 	}
 	eq->elevator_data = nd;
 
+	add_queue(q);
```

Comment:
there is an add_queue call but no del_queue - it's seems to be a leak, isn't it?